recipes/vulkan-loader/all/conanfile.py

```diff
@@ -0,0 +1,94 @@
+import os
+import glob
+from conans import ConanFile, tools, CMake
+
+
+class VulkanLoaderConan(ConanFile):
+    name = "vulkan-loader"
+    description = "Khronos official Vulkan ICD desktop loader for Windows, Linux, and MacOS."
+    topics = ("conan", "vulkan", "loader", "desktop", "gpu")
+    homepage = "https://github.com/KhronosGroup/Vulkan-Loader"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "Apache-2.0"
+    exports_sources = ["CMakeLists.txt"]
+    settings = "os", "arch", "build_type", "compiler"
+    generators = "cmake", "cmake_find_package"
+    short_paths = True
```

Comment:
```suggestion
```

Reply:
short_paths is required for Windows (See https://github.com/conan-io/conan-center-index/pull/2581#issuecomment-693785014)

Reply:
see this comment https://github.com/conan-io/conan-center-index/pull/2581#discussion_r490600643 ;)
It might not work, but give it a try please :)